### PR TITLE
add hswish and hsigmoid

### PR DESCRIPTION
**Pull request by louzana**

### CODE REVIEW

File: mmcv/cnn/bricks/hsigmoid.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    Hsigmoid(x) = min(max(0, (x + 1) / 2), 1)
+
+    Returns:
+        Tensor: The output tensor.
+    """
+
+    def __init__(self):
+        super(HSigmoid, self).__init__()
+
+    @staticmethod
```
Comment (hellock) [inline mmcv/cnn/bricks/hsigmoid.py:18]: Why staticmethod?

Comment (louzana) [reply mmcv/cnn/bricks/hsigmoid.py]: It's useless. Deleted in the new commit.

File: mmcv/cnn/bricks/hsigmoid.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    Hsigmoid(x) = min(max(0, (x + 1) / 2), 1)
+
+    Returns:
+        Tensor: The output tensor.
+    """
+
+    def __init__(self):
+        super(HSigmoid, self).__init__()
+
+    def forward(x):
```
Comment (hellock) [inline mmcv/cnn/bricks/hsigmoid.py:18]: `forward(self, x)`

**Comment by hellock** [comment]
Add the corresponding unittests.

File: tests/test_cnn/test_hsigmoid.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    input = torch.randn(input_shape)
+    output = act(input)
+    expected_output = torch.min(
+        torch.max((input + 1) / 2, torch.zeros(input_shape)),
+        torch.ones(input_shape))
+    # test output shape
+    assert output.shape == expected_output.shape
+    # test output value
+    assert np.equal(output.cpu().data.numpy(),
+                    expected_output.cpu().data.numpy()).all()
```
Comment (hellock) [inline tests/test_cnn/test_hsigmoid.py:19]: `torch.equal()` is simpler.

**Comment by openmmlab-v** [comment]
Hi @louzana ！First of all, we want to express our gratitude for your significant PR in the mmcv project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

If you are Chinese or have WeChat，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**